### PR TITLE
[1LP][RFR] Put in order storage tests

### PR DESCRIPTION
**Pull request by mmojzis**
Deleted duplicate tests from test_manual.py
Reassigned several tests
Moved some tests from test_manual.py
@apagac Why were some storage snapshot tests assigned to you?

### CODE REVIEW

**Comment by apagac** [comment]
Seems like it happened because @mshriver wanted to assign all testcases to FA owners, and he saw the string 'snapshot' so it landed on me :)

**Comment by mshriver** [comment]
@mmojzis @apagac That's correct - some tests hit two different focus areas, and I just picked one when making assignee required. Thanks for re-assigning!

File: cfme/tests/storage/test_volume.py [hunk 1]
```diff
@@ -157,6 +159,25 @@ def test_storage_volume_crud(appliance, provider, from_manager):
                                unblock=lambda provider: provider.one_of(EC2Provider))])
 @pytest.mark.tier(1)
 def test_storage_volume_attach_detach(appliance, provider, instance_fixture, from_manager):
+    """ Test storage volume attach/detach
+
+    prerequisites:
+        * Storage provider
+        * Instance
+
+    Steps:
```
Comment (izapolsk) [inline cfme/tests/storage/test_volume.py]: please move under Polarion section following our parsing agreements.

File: cfme/tests/storage/test_volume.py [hunk 2]
```diff
@@ -173,6 +194,32 @@ def test_storage_volume_attach_detach(appliance, provider, instance_fixture, fro
     volume.delete()
 
 
+@pytest.mark.manual
+@test_requirements.storage
+def test_storage_volume_attached_delete():
+    """
+    Requires:
+    RHCF3-21779 - test_storage_volume_attach[openstack]
+    Steps to test:
```
Comment (izapolsk) [inline cfme/tests/storage/test_volume.py]: same thing

File: cfme/tests/storage/test_volume.py (hunk 1 above)
Comment (mmojzis) [reply cfme/tests/storage/test_volume.py]: Sorry, I don't understand. Can you please give me an example?
Thanks

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/storage/test_volume.py (hunk 1 above)
Comment (mmojzis) [reply cfme/tests/storage/test_volume.py]: Both fixed

File: cfme/tests/storage/test_object_store_container.py [hunk 3]
```diff
@@ -50,3 +51,29 @@ def test_add_remove_tag(container):
...
+        assignee: anikifor
+        caseimportance: medium
+        casecomponent: Cloud
+        initialEstimate: 1/8h
+        startsin: 5.7
+
+        Requires:
+            OpenstackProvider
+
+        Steps:
```
Comment (mshriver) [inline cfme/tests/storage/test_object_store_container.py]: This needs to be `testSteps`, and please put `expectedResults` as well, splitting what's here between the two.

Comment (mshriver) [reply cfme/tests/storage/test_object_store_container.py]: Apply to all below. Please see `conf/polarion_tools.yaml` for the keys, or other tests for examples.

Comment (mmojzis) [reply cfme/tests/storage/test_object_store_container.py]: Added for manual tests.
But now I don't actually know what to do when there is multiple steps test and after which test which result is expected.
For automated tests I didn't add expectedResults as I don't think it's needed when it's all documented in the code? @mshriver

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/storage/test_object_store_container.py (hunk 3 above)
Comment (mshriver) [reply cfme/tests/storage/test_object_store_container.py]: We don't require testSteps/expectedResults for automation - Just inline comments, and comments as to the content of the test in the regular docblock, out of the Polarion section.

I don't quite follow your other concern, but I think you might be getting at not having expected results for some teststeps.  You can do the following:

```
testSteps:
  1. step one
  2. step two
  3. step three
expectedResults:
  1. thing happens
  2. 
  3. other thing happens
```

Comment (mmojzis) [reply cfme/tests/storage/test_object_store_container.py]: So expectedResults number should be same number as the step where do I expect that result? So I can have for example 10 steps, but I want expectedResult only for step 7 and 10 so in expected results I will have
```
expectedResults:
    7. something is equal something
    10. something is deleted
```
Like this?

Comment (mshriver) [reply cfme/tests/storage/test_object_store_container.py]: 95% correct - you still need to have empty `1.`, `2.`, etc

Comment (mmojzis) [reply cfme/tests/storage/test_object_store_container.py]: @mshriver Reworked it I hope correctly.

File: cfme/tests/storage/test_volume.py [hunk 4]
```diff
@@ -157,6 +157,25 @@ def test_storage_volume_crud(appliance, provider, from_manager):
...
 def test_storage_volume_attach_detach(appliance, provider, instance_fixture, from_manager):
+    """ Test storage volume attach/detach
+    Polarion:
+        assignee: mmojzis
+        initialEstimate: 1/4h
+        casecomponent: Cloud
+        startsin: 5.7
+        caseimportance: high
+
+        prerequisites:
```
Comment (mshriver) [inline cfme/tests/storage/test_volume.py]: Two related things here:
Polarion flake parser sees the empty line as the end of the polarion metadata section.

1. prerequisites isn't a valid polarion key, if you want to include it in polarion , it should be `'setup:'`.
2. testSteps should not have an empty line before it, it needs to immediately follow caseimportance (or whatever previous polarion metadata line may be)

File: cfme/tests/storage/test_object_store_container.py (hunk 3 above)
Comment (mshriver) [reply cfme/tests/storage/test_object_store_container.py]: Looking much better, thanks!! One last comment about empty lines in the docblocks and you'll be good to go.

**Pull request merged**